eras/shelley/impl/cardano-ledger-shelley.cabal

```diff
@@ -55,7 +55,7 @@ library
     Cardano.Ledger.Shelley.PoolRank
     Cardano.Ledger.Shelley.PParams
     Cardano.Ledger.Shelley.Rewards
-    Cardano.Ledger.Shelley.RewardProvenance
+    Cardano.Ledger.Shelley.RewardProvenance    
```

Comment:
trailing whitespace (I _think_ ormolu will notice as well)

Reply:
Ormolu does not touch cabal files